go-update/src/main/java/org/reactome/release/goupdate/DuplicateReporter.java

```diff
@@ -86,13 +86,21 @@ public DuplicateReporter(MySQLAdaptor adaptor)
 				{
 					long dbid = i.getDBID();
 					int refCount = getReferrerCountforInstance(i, classesToIgnore);
-					referrerCounts.put(dbid,refCount);
+					referrerCounts.put(dbid, refCount);
 				}
 			}
 		}
 		return referrerCounts;
 	}
 	
+	// TODO: Move this function to release-common-lib, maybe. It's a generic utility function, could be useful somewhere else.
```

Comment:
I can't off the top of my head think of any modules that try to get a referrer count so it might be OK here